### PR TITLE
Add 2020 Q4 financial transparency report

### PR DESCRIPTION
**Pull request by lehnberg**
`ref_balance` checks out:
```
{
      "tx_hash": "5c75c669f4808b88a151b8e867a5aede075a0611f294c27365dd5b1fa9856913",
      "block_height": 662519,
      "tx_input_n": 0,
      "tx_output_n": -1,
      "value": 1000000000,
      "ref_balance": 9687264400,
      "confirmations": 12058,
      "confirmed": "2020-12-22T17:18:29Z",
      "double_spend": false
    }
```

https://api.blockcypher.com/v1/btc/main/addrs/bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx

### CODE REVIEW

File: financials/reports/funding_transparency_2020Q4.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+### Spent
+
+During Oct 01 - Dec 30 2020, Grin spent the following assets:
+
+Currency | Amount | Tx fees | Total amount |
+|---|---:|---:|---:|
+BTC|7.638498|0.002864|7.641362
+
+### Comments
+* Tx outputs were split into smaller 10BTC chunks in this period. Aside from the transaction fees involved in doing so, all spending in the period was related to funding requests, specifically:
```
Comment (DavidBurkett) [inline financials/reports/funding_transparency_2020Q4.md:48]: Where is Antioch's Q4 2020 payment? I don't see it here or on the Q3 report: https://github.com/mimblewimble/grin-pm/blob/5d9578efe2d889d84293f4c1f99ce7ce6f926544/financials/reports/funding_transparency_2020Q3.md#activity-during-the-period

Comment (DavidBurkett) [reply financials/reports/funding_transparency_2020Q4.md:48]: It looks like your Q4 payment is not listed either.

Comment (lehnberg) [reply financials/reports/funding_transparency_2020Q4.md:48]: Thanks for raising, was included in the numbers, but was missed in the call outs. Fixed.

**Review by DavidBurkett — APPROVED** [review]

File: financials/reports/funding_transparency_2020Q4.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Status before the period
+
+### Disposable funds
+
+At the end of Sep 30 2020, Grin held disposable funds to the equivalent of $1,127,130.25 in the following assets:
+
+Currency | Amount | USD x-rate Sep 30 2020 | USD Equivalent | Wallet address(es)
+|---|---:|---:|---:|---|
+GRIN | 3770.71555489 | $0.306694 | $1,156.46 | https://donations.grin-tech.org
```
Comment (l33d4n) [inline financials/reports/funding_transparency_2020Q4.md:21]: https://donations.grin-tech.org is expired and currently used as a spam website.

**Review by l33d4n — CHANGES_REQUESTED** [review]

File: financials/reports/funding_transparency_2020Q4.md (hunk 2 above)
Comment (lehnberg) [reply financials/reports/funding_transparency_2020Q4.md:21]: Good point. This should probably be addressed across all the old reports, and could then be raised in a distinct pull request. I've filed an issue to track this: https://github.com/mimblewimble/grin-pm/issues/406

**Pull request merged**